admyral/db/admyral_store.py

```diff
@@ -232,6 +232,15 @@ async def store_action(self, action: PythonAction) -> None:
 
             await db.commit()
 
+    async def delete_action(self, action_type: str) -> None:
+        async with self._get_async_session() as db:
+            await db.exec(
+                delete(PythonActionSchema).where(
+                    PythonActionSchema.action_type == action_type
+                )
```

Comment:
You need to consider the `user_id` too. Can you merge the newest changes into your branch.